photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Params.scala

```diff
@@ -14,23 +14,24 @@
  */
 package com.linkedin.photon.ml.cli.game.training
 
+import com.linkedin.photon.ml.OptionNames._
+import com.linkedin.photon.ml.cli.game.FeatureParams
 import com.linkedin.photon.ml.optimization.game.{MFOptimizationConfiguration, GLMOptimizationConfiguration}
 import com.linkedin.photon.ml.data.{FixedEffectDataConfiguration, RandomEffectDataConfiguration}
 import com.linkedin.photon.ml.io.ModelOutputMode
 import com.linkedin.photon.ml.io.ModelOutputMode._
 import com.linkedin.photon.ml.supervised.TaskType
 import com.linkedin.photon.ml.supervised.TaskType._
+import com.linkedin.photon.ml.util.PalDBIndexMapParams
 import scopt.OptionParser
 
-import scala.collection.{Map, Set}
-
 /**
  * A bean class for GAME training parameters to replace the original case class for input parameters.
  * @note Note that examples of how to configure GAME parameters can be found in the integration tests for the GAME
  *       driver.
  * @todo Making the way GAME being configured more user friendly
  */
-class Params {
+class Params extends FeatureParams with PalDBIndexMapParams {
```

Comment:
That's right, @timyitong. This way we can group the related Params together, and pass in an object of type ```IndexMapParams``` object into the ```IndexMapLoader``` instead of the app-global ```Params``` type.